bnr-utils/nz_azConnector/nz_azConnector.go

```diff
@@ -301,9 +311,61 @@ func (cn *Conn) downloadBkp(outdir string, uniqueid string, blobpath string, str
     close(work)
     <- done
     log.Println("Total files downloaded:", filesdownloaded)
+    updateLocation(locations,outdir)
+    updateContents(contents)
     return err
 }
 
+func updateLocation(arrLoc []string,outdir string){
+    for _,locFile := range arrLoc{
+        input, err := ioutil.ReadFile(locFile)
+        if err != nil {
+            log.Fatalf("Unable to open %s to read: %v\n", locFile, err)
+        }
+        lines := strings.Split(string(input), "\n")
+        if (len(lines) == 2 && !strings.HasSuffix(lines[len(lines) -2] , outdir)) {
+            f, err := os.OpenFile(locFile,
+                os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+            if err != nil {
+                log.Fatalf("Unable to open %s for update: %v\n", locFile, err)
+            }
+            defer f.Close()
+            textAppend := "1,1,1," + outdir + "\n"
+            if _, err := f.WriteString(textAppend); err != nil {
+                log.Fatalf("Unable to update %s: %v\n", locFile, err)
+            }
+        }
+    }
+}
+
+func updateContents(arrContents []string){
+    for _,contentFile := range arrContents{
+        input, err := ioutil.ReadFile(contentFile)
+        if err != nil {
+            log.Fatalln("Unable to open %s to read: %v\n",contentFile,err)
+        }
+
+        lines := strings.Split(string(input), "\n")
+        var textline []string
+        for i := 0 ; i < len(lines) ; i++ {
+            line := lines[i]
+            token := strings.Split(line, ",")
+            if ( token[len(token)-1] == "0" ) {
+                r := []rune(line)
+                str := string(r[:len(r)-1]) + "1"

```

Comment:
I still think we don't need `rune`. Pl consider changing to

```go
for _, line := range lines {
    token := strings.Split(line, ",")
    if token[len(token)-1] == "0" {
        token[len(token)-1] = "1"
    }
    textline = append(textline, strings.Join(token, ","))
}
```

Reply:
Removed rune and done changes as per suggestion